improc/zogy.py

```diff
@@ -224,7 +255,23 @@ def zogy_subtract(image_ref, image_new, psf_ref, psf_new, noise_ref, noise_new,
     alpha = np.fft.fftshift(alpha)
     alpha_std = np.fft.fftshift(alpha_std)
 
-    return sub_image, sub_psf, score, score_corr, alpha, alpha_std
+    translient = np.fft.fftshift(Z)
+    translient_sigma = norm.isf(chi2.sf(translient, df=2))
+    translient_corr = np.fft.fftshift(Z_corr)
+    translient_corr_sigma = norm.isf(chi2.sf(translient_corr, df=2))
+
+    return dict(
```

Comment:
If I'm not mistaken, this is also going to be the return of pipeline.subtraction._subtract_zogy (with some modifications).  The docstring there should be updated to reflect this.

Do you use this translient score anywhere yet?  I don't think so.  We'll need to be careful about assuming that there are things in the subtraction output that are specific to certain algorithms.  We may have to adjust our scoring of transients based on what is there and/or what the subtraction method is.

Reply:
Yes, we will need to always check the dictionary to see if we have the things we want.